### DISCUSSION TITLE
Couple of questions

### DISCUSSION THREAD

**Discussion by blahdy** [creepymonster/GlucoseDirect#393] [Q&A] [answered]
Just found your project, couldn’t find the answers. 

Can your app activate libre 1 sensors?

Does it support US 14 days libre 1?

**Comment by creepymonster** [comment, marked as answer]
Hello,

thank you very much for your message.

You cannot activate the Libre 1 sensor with GD. This is a process that I have always done with LibreLink. I was not sure if with an error in the activation perhaps a sensor could be destroyed. The problem can not occur when reading out. Therefore I have not installed the activation.

The LibreUS14 sensors are supported. I myself have no such sensors - but have from another user the readout has worked.

Many greetings
Reimar

PS. If you have problems with just send a logfile ☺️

**Reply by blahdy** [reply]
Thank you very much for such a quick answer. The question comes from searching an alternative to the abandoned diabox and your app looks much better in terms of UI. 

Just hoped for single, hassle free solution for an elder relative. 

Hope you’ll implement this functionality one day.

**Reply by blahdy** [reply]
Your app was, btw, proposed as the alternative by the diabox author in his telegram channel when his was taken down.

**Reply by creepymonster** [reply]
I was not aware of that, thanks for the info. I have used DiaBox for a long time very gladly.

For the US14 sensor, I'm sure you have a bubble sensor - they are also supported by me. Because as far as I know, the US14 sensor has no Bluetooth otherwise.

Many greetings
Reimar

**Reply by Mrrombout** [reply]
Sorry to bother you with 🎄. I am looking for a link to download Glucose Direct. Do you have an address? Or Link? Thnk you.

**Reply by hestovall** [reply]
I’m also looking to use Glucose Direct and would like a download link if possible! Thanks!
